### DISCUSSION TITLE
Using PyODBC with Databricks to write pandas dataframes

### DISCUSSION THREAD

**Discussion by Rhiyo** [mkleehammer/pyodbc#1350] [Q&A]
In the databricks documentation it suggests using pyodbc for reading and shows instructions on how to do so: https://docs.databricks.com/en/dev-tools/pyodbc.html

However, I've yet to find a good solution for writing. The best examples I can find for easily writing pandas tables to a server using pyodbc involve sqlachemy and only seem to work with mssql dialect (connection string with mssql+pyodbc).

I could do insert queries directly with pyodbc - but it's quite annoying without ORM support as I want to do it for arbitrary dataframes. Does anyone have any ideas on how I could achieve this with ease?

**Comment by gordthompson** [comment]
https://docs.databricks.com/en/dev-tools/sqlalchemy.html
